docs/docsite/rst/roadmap/COLLECTIONS_3_0.rst

```diff
@@ -0,0 +1,45 @@
+===================
+Ansible project 3.0
+===================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-base <https://pypi.org/project/ansible-base/>`_ package as well. All dates are subject to change. See :ref:`base_roadmap_2_11` for the most recent updates on ``ansible-base``.
+
+.. contents::
+   :local:
+
+Release schedule
+=================
+
+.. note::
+
+    Ansible its switching from its traditional versioning scheme to `semantic versioning <https://semver.org/>`_ for this release.  So the version will be 3.0.0 instead of 2.11.0.
+
+
+
+:2020-12-16: Finalize rules for net-new collections submitted for the ansible release.
+:2021-01-27: Final day for new collections to be **reviewed and approved**. They need to have been
+    submitted prior to this to give reviewers a chance to look them over and for collection owners
+    to fix any problems.
+:2021-02-02: Ansible-3.0.0-beta1 -- feature freeze [1]_
+:2021-02-09: Ansible-3.0.0-rc1 -- final freeze [2]_ [3]_
+:2021-02-16: Release of Ansible-3.0.0
+:2021-03-09: Release of Ansible-3.1.0 (bugfix + compatible features: every three weeks)
+
+.. [1] No new modules or major features accepted after this date. In practice this means we will freeze the semver collection versions to compatible release versions. For example, if the version of community.crypto on this date was community-crypto-2.1.0; ansible-3.0.0 could ship with community-crypto-2.1.1.  It would not ship with community-crypto-2.2.0.
+
+.. [2] After this date only changes blocking a release are accepted.  Changes require a new rc to be made and may slip the final release date.
+.. [3] Collections will only be updated to a new version if a blocker is approved.  Collection owners should discuss any blockers at a community IRC meeting (before this freeze) to decide whether to bump the version of the collection for a fix. See the `Community IRC meeting agenda <https://github.com/ansible/community/issues/539>`_.
+
+
+.. note::
+
+  Breaking changes may be introduced in Ansible 3.0.0, although we encourage collection owners to use deprecation periods that will show up in at least one Ansible release before being changed incompatibly.
+
+
+Ansible patch releases
+=======================
+
+Ansible 3.0.x patch releases will occur approximately every three weeks if changes to collections have been made or if it is deemed necessary to force an upgrade to a later ansible-base-3.0.x.  Ansible 3.0.x patch releases may contain new features but not backwards incompatibilities.  In practice, this means we will include new collection versions where either the patch or the minor version number has changed but not when the major number has changed. For example, if Ansible-3.0.0 ships with community-crypto-2.1.0; Ansible-3.1.0 may ship with community-crypto-2.2.0 but would not ship with community-crypto-3.0.0).
```

Comment:
```suggestion
Ansible 3.0.x patch releases will occur approximately every three weeks if changes to collections have been made or if it is deemed necessary to force an upgrade to a later ansible-base-2.10.x.  Ansible 3.0.x patch releases may contain new features but not backwards incompatibilities.  In practice, this means we will include new collection versions where either the patch or the minor version number has changed but not when the major number has changed. For example, if Ansible-3.0.0 ships with community-crypto-2.1.0; Ansible-3.1.0 may ship with community-crypto-2.2.0 but would not ship with community-crypto-3.0.0).
```

Reply:
I'm confused by line 42.  Since it focuses on patch releases - Ansible 3.0.x, I'm unclear what the last 'for example' means since it then talks about Ansible 3.1.0. (minor version release, not a patch release).  So with semver - are the 3-week releases patches (3.0.1, 3.0.2) or are they minor version releases (3.1.0, 3.2.0 etc)?  OR can they be either, depending on what happens with the collections underneath?

Reply:
@samccann Technically they could be either one.  My plan, though, was to just use minor versions for all of them.  If I can automate examining the versions of collections being updated to tell when they are all patch releases, themselves, I could make the release number a patch release vs minor release depending on whether the underlying collections claim to have added new features.  I'm not sure whether that will make end users happier (They can know which ansible releases add features) or whether it will make them unhappier (it's no longer possible to predict what the next release number will be), though.

Thoughts?  Should I bring the release number question to the Community Meeting in a few weeks or just go ahead with the plan to always use minor version bumps?

Reply:
ok tried again to reword it below. Basically let's just call them Ansible minor versions since that's what we are going with and patch means something specific in semver, and we aren't doing patches at the moment.

We could always say patches are reserved for the oopsies like we had with 2.10.3 going to 2.10.4 pretty quickly.